core/connection/manager.go

```diff
@@ -51,13 +52,18 @@ var (
 // consumer identity, provider identity and uses state channel to report state changes
 type ConnectionCreator func(ConnectOptions, StateChannel) (Connection, error)
 
+type sessionStorageSave interface {
```

Comment:
Can we name this interface as `sessionSaver` to follow golang interface naming recommendations?

Reply:
Will rename it in next PR